src/graphql/resolvers/box-resolver.ts

```diff
@@ -125,6 +125,39 @@ export class BoxResolver {
     @Ctx() context: GraphQLContext,
     @Info() info: GraphQLResolveInfo
   ) {
+    const arrayArguments = [addresses, boxIds, ergoTrees];
+    let arrayArgumentsLength = 0;
+    for (const arg of arrayArguments) {
+      if (arg) {
+        arrayArgumentsLength += arg.length;
+      }
+    }
+
+    const literalArguments = removeUndefined({
```

Comment:
You can simplify this, by checking all arguments at once, like you are doing here.

Another good thing to have, to keep code as clean as possible, is to move this verification to a private method, something like `_countValidArguments(): number`